mappings/net/minecraft/network/encryption/ChatMessageSignature.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_7469 net/minecraft/network/encryption/ChatMessageSignature
```

Comment:
Recommended rename to `MessageSignature` in line with `MessageSender`, `MessageType`

Reply:
Unsure, the classes mentioned I believe are used by non chat messages too, this one is not so.